docs/sails-js/client-generation.md

```diff
@@ -0,0 +1,276 @@
+---
+sidebar_position: 3
+sidebar_label: Client Generation
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Client Generation
+
+The `sails-js-cli` is a command-line tool designed to generate TypeScript client libraries from Sails IDL files. It automates the creation of fully functional client libraries based on the interfaces defined in the Sails framework, streamlining development and ensuring consistency between client and on-chain programs.
```

Comment:
I think it's worth to attach a link to the `Overview` section with the specifics of the cli over raw sails.